src/metabase/query_processor/store.clj

```diff
@@ -114,7 +114,7 @@
   [database-id-or-metadata-providerable :- ::database-id-or-metadata-providerable]
   (if (integer? database-id-or-metadata-providerable)
     (lib.metadata.jvm/application-database-metadata-provider database-id-or-metadata-providerable)
-    database-id-or-metadata-providerable))
+    (lib.metadata/->metadata-provider database-id-or-metadata-providerable)))
```

Comment:
Small bug with the QP store, it said `or-metadata-providerable` but it didn't actually let you pass something random that you can get a metadata provider from like a query. Fixed